src/common/Makefile

```diff
@@ -94,6 +94,11 @@ OBJS_COMMON += \
 	sha2.o
 endif
 
+ifeq ($(with_libseccomp),yes)
```

Comment:
src/common is intended for code that's used in both the backend and frontend. Not sure where we should stick `seccomp.c`, but not in src/common. Maybe in src/backend/postmaster/ ?

Reply:
> Maybe in src/backend/postmaster/

I don't mind :)

Reply:
Done.